src/commons/sideContent/content/SideContentAutograder.tsx

```diff
@@ -81,6 +90,12 @@ const SideContentAutograder: React.FC<SideContentAutograderProps> = props => {
 
   return (
     <div className="Autograder">
+      {isFolderModeEnabled && (
+        <span className="entrypoint-span">
+          Entry point file for Testcases:{' '}
+          <code className="filepath-design bp5-code">{currentFileBeingRan}</code>
```

Comment:
Don't use raw `bp5-` classes. Use the abstracted `Classes` object instead. This will allow us to change the underlying CSS namespace without changing the code.